source/fundamentals/builders/aggregation-expression-operations.txt

```diff
@@ -150,68 +153,66 @@ You can perform an arithmetic operation on a value of type ``MqlInteger`` or
      - :manual:`$divide </reference/operator/aggregation/divide/>`
 
-   * - `multiply() for MqlInteger <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlInteger.html#multiply(int)>`__
+   * - | `multiply() for MqlInteger <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlInteger.html#multiply(int)>`__
        | `multiply() for MqlNumber <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#multiply(com.mongodb.client.model.mql.MqlNumber)>`__
+
      - :manual:`$multiply </reference/operator/aggregation/multiply/>`
 
    * - `round() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#round()>`__
      - :manual:`$round </reference/operator/aggregation/round/>`
 
-   * - `subtract() for MqlInteger <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlInteger.html#subtract(int)>`__
+   * - | `subtract() for MqlInteger <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlInteger.html#subtract(int)>`__
        | `subtract() for MqlNumber <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#subtract(com.mongodb.client.model.mql.MqlNumber)>`__
-     - :manual:`$subtract </reference/operator/aggregation/subtract/>`
 
-.. include:: /includes/fundamentals/agg-mql-diff.rst
+     - :manual:`$subtract </reference/operator/aggregation/subtract/>`
 
-Suppose you need to calculate the total cost of a sale from the item's
-price, the quantity sold, and an applicable discount.
+Suppose you have weather data for a specific year that includes the
+precipitation measurement (in inches) for each day. You want find the average
+precipitation, in millimeters, for each month.
 
-The ``multiply()`` operator multiples the ``itemPrice`` and the ``quantity``
-to calculate the initial amount for the sale. The ``subtract()`` operator
-deducts the ``discount`` from the initial sale value to evaluate the
-total value.
+The ``multiply()`` operator multiplies the ``precipitation`` field by
+``25.4`` to convert the value to millimeters. The ``avg()`` accumulator method
+returns the average as the ``avgPrecipMM`` field. The ``group()`` method
+groups the values by month.
 
-The following code shows the pipeline for this aggregation and assigns
-cost of the sale to a new field ``totalSale``:
+The following code shows the pipeline for this aggregation:
 
 .. code-block:: java
    :copyable: true
 
-   asList(project(fields(
-            computed("totalSale",
-               current()
-                  .getInteger("itemPrice")
-                  .multiply(current()
-                     .getInteger("quantity"))
-                  .subtract(current()
-                     .getInteger("discount")))
-   )));
+   var precip = current().getInteger("precipitation");
+
+   asList(group(
+       "$month",
```

Comment:
Does this accept something like `var date = current().getDate("$date")` `date.getMonth()` instead of `$month`? This would demonstrate an expression in the group here.